pallets/foreign-investments/src/impls/mod.rs

```diff
@@ -325,6 +328,10 @@ impl<T: Config> Pallet<T> {
 		match state {
 			InvestState::NoState => {
 				InvestmentState::<T>::remove(who, investment_id);
+				#[cfg(feature = "std")] {
+					println!("Clearing InvestState");
+				}
```

Comment:
Of curiosity, is it possible to use the `log` instead so that we can have these logs both on std and wasm?

Reply:
Sorry, this is an unwanted remnant from debugging integration tests. I will remove it!